doc/nrf/releases_and_maturity/releases/release-notes-2.5.0.rst

```diff
@@ -0,0 +1,1304 @@
+  * ``#XGPS`` AT command to control the GNSS module with support for A-GNSS and P-GPS at the same time.
+
+* Updated:
+
+  * The configuration to enable support for nRF Cloud A-GNSS service and nRF Cloud Location service by default.
+  * UART receive refactored to utilize hardware flow control (HWFC) instead of disabling and enabling UART receiving between commands.
+  * UART transmit has been refactored to utilize buffering.
+    Multiple responses can now be received in a single transmission.
+  * Modem FOTA to only need a modem reset to apply the firmware update.
+    The full chip reset (using the ``#XRESET`` AT command) remains supported.
+  * ``#XGPSDEL`` AT command to disallow deleting local clock (TCXO) frequency offset data because it is an internal value that should not be deleted when simulating a cold start.
+  * Socket option ``TLS_DTLS_HANDSHAKE_TIMEO`` to a new name value.
+  * ``#XTCPSVR`` connection closure status and documentation.
+  * ``#XRECVFROM`` to include the port of the peer.
+
+* Removed:
+
+  * DFU AT commands ``#XDFUGET``, ``#XDFUSIZE`` and ``#XDFURUN`` because they were not usable without a custom application in the target (nRF52 Series) device.
+  * Support for bootloader FOTA update because it is not needed for Serial LTE modem.
+  * Operations to read or erase the MCUboot secondary slot from the ``#XFOTA`` AT command because the application update process overwrites the slot in any case.
+  * AT commands ``#XCELLPOS`` and ``#XWIFIPOS``.
+    They are replaced by the ``#XNRFCLOUDPOS`` command that allows to combine cellular and Wi-Fi data to determine the device location.
+  * The AT commands ``#XAGPS`` and ``#XPGPS``.
+    Their functionality is merged into the ``#XGPS`` AT command that now allows using A-GNSS and P-GPS at the same time.
+  * The AT command ``#XSLMUART``.
+    UART is now configured using only devicetree.
+
+nRF5340 Audio
+-------------
+
+* Modified the entire application architecture for handling Bluetooth LE Audio.
+  The following new modules have been added:
+
+  * Management - This module handles scanning and advertising, in addition to general initialization, controller configuration, and transfer of DFU images.
+    The new architecture makes it possible to make connections and handle periodic advertising sync independently of the Bluetooth LE Audio setup.
+  * Stream - This module handles the setup and transfer of audio in the Bluetooth LE Audio context.
+    The new architecture makes it possible to have more than one Bluetooth LE Audio role in one device.
+  * Renderer - This module handles rendering, such as volume up and down.
+  * Content Control - This module handles content control, such as play and pause.
+
+* Added back the QDID number for the :ref:`lib_bt_ll_acs_nrf53_readme` to the documentation.
+* Updated the :ref:`application documentation <nrf53_audio_app>` by splitting it into several pages.
+* Fixed a compatibility issue when using Google Pixel 7/7A.
```

Comment:
```suggestion

```
We still miss one config for compatibility with Google Pixel 7/7A. so we have to remove this point for now

Reply:
This was discovered today